appium/webdriver/common/appiumby.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium.webdriver.common.by import By
+
+
+class AppiumBy(By):
```

Comment:
I'd prefer it to not be a breaking change. E.g. keep the MobileBy strategy, but simply mark it as deprecated
AppiumBy may then be inherited from MobileBy to avoid unnecessary duplication.

Reply:
`MobileBy.ID` in appium/webdriver/common/mobileby.py (inherits AppiumBy) etc are `str`. `AppiumBy.ID` also `str`. So basically both ID are set as `id` to find_element, for example.
I think they do not have breaking change... something wrong?

Yea, I was finding the way to print deprecation message when `MobileBy.ID` was called or when `MobileBy.ID` was given in find_element. But it was simply `str`.
So currently I've put the deprecation message in the module description.

Reply:
Maybe we could use https://www.py4u.net/discuss/182959 (https://pypi.org/project/deprecation/#description) to mark things as deprecated

Reply:
Also https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated

Reply:
mm, the deprecation package also does not work for str of class variables?

Yea, I think only documentation can mention it.